portfolio/src/main/webapp/script.js

```diff
@@ -44,3 +53,21 @@ function deletePost() {
 
   fetch(url, {method: 'POST'}).then(result => populatePosts());
 }
+
+/**
+  * Updates the score of a post in Datastore and on the DOM.
+  */
+function votePost(voteButton, direction) {
+  var score;
+  if (direction==1) {
+    score = voteButton.previousElementSibling;
+  } else {
+    score = voteButton.nextElementSibling;
+  }
+
+  const comment = voteButton.parentElement.parentElement.firstChild.nextElementSibling.textContent;
```

Comment:
Use a querySelector to get this.

Reply:
I believe it isn't possible to use querySelector because the posts will not have a unique class/id. The way I have it setup is by passing 'this' as a parameter to the function.